platform/server/elasticsearch/Cluster.ts

```diff
@@ -13,7 +13,7 @@ export class Cluster {
     private readonly config: ElasticsearchConfig,
     logger: LoggerFactory
   ) {
-    this.log = logger.get('elasticsearch', 'client', config.clusterType);
+    this.log = logger.get(`elasticsearch::client::${config.clusterType}`);
```

Comment:
IIRC it was mostly because of string keys in our internal maps where we store appenders/loggers, but I'll give it another try now when I have everything in place.